StabilityClient.Net/StabilityClient.Net/Models/GenerateResponseSaveResult.cs

```diff
@@ -0,0 +1,5 @@
+namespace StabilityClient.Net.Models; 
+
+public class GenerateResponseSaveResult {
+    public string FullPath { get; set; } = "";
+}
```

Comment:
Please add missing endlines to all files ;)

Reply:
Ok, I added :)